File: official/docs/php/customs_info/CreateCustomsInfo.php
<?php

\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$customsItem = \EasyPost\CustomsItem::create(...);

$customsInfo = \EasyPost\CustomsInfo::create([
    "eel_pfc" => 'NOEEI 30.37(a)',
    "customs_certify" => true,
    "customs_signer" => 'Steve Brule',
    "contents_type" => 'merchandise',
    "contents_explanation" => '',
    "restriction_type" => 'none',
    "customs_items" => [$customsItem, [
        "description" => 'T-shirt',
        "quantity" => 1,
        "weight" => 5,
        "value" => 10,
        "hs_tariff_number" => '123456',
        "origin_country" => 'US'
    ]]
]);

echo $customsInfo;

File: official/docs/go/customs_info/create_customs_info.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	customsInfo, err := client.CreateCustomsInfo(
        &easypost.CustomsInfo{
            CustomsCertify: true,
			CustomsSigner: "Steve Brule",
			ContentsType: "merchandise",
			ContentsExplanation: "",
			RestrictionType: "none",
            EELPFC:         "NOEEI 30.37(a)",
            CustomsItems: []*easypost.CustomsItem{
                &easypost.CustomsItem{
                    Description:   "T-shirt,
                    Quantity:      1,
                    Value:         10.00,
                    Weight:        5,
                    OriginCountry: "US",
                },
            },
        },
    )
	fmt.Println(customsInfo) 

}
